### ISSUE TITLE
Is there a calculation code for LV loss?

### ISSUE THREAD

**Issue by zhouke0304** [HUANGLIZI/LViT#27]
I did not find the calculation code related to LV loss and EPI in the code, may I ask how to train under semi-supervised conditions?

**Comment by HUANGLIZI** [comment]
Thanks for your message. You can refer to the COSINESIMILARITY function. https://pytorch.org/docs/stable/generated/torch.nn.CosineSimilarity.html
